apps/pkt-gen/pkt-gen.c

```diff
@@ -1792,6 +1925,14 @@ receiver_body(void *data)
 			goto quit;
 		}
 #endif /* !BUSYWAIT */
+		if (targ->garp) {
+			clock_gettime(CLOCK_REALTIME_PRECISE, &tmptime);
```

Comment:
Same as above

Reply:
I'm just going along with what was already being used.

Reply:
Yes, but here you just need to know when 1 sec has passed. You can't use COARSE actually, because it's Linux specific, sorry.
So we can just use CLOCK_REALTIME.

Reply:
Sorry, I had not seen that the _PRECISE is just a macro that maps back to CLOCK_REALTIME, which is ok.

Reply:
Aha, you just beat me to saying that